js/theme.js

```diff
@@ -0,0 +1,38 @@
+define([
+  "core/js/adapt",
+  "./themePageView",
+  "./themeArticleView",
+  "./themeBlockView",
+  "./themeView"
+], function(Adapt, ThemePageView, ThemeArticleView, ThemeBlockView, ThemeView) {
+
+  function onDataReady() {
+    $("html").addClass(Adapt.course.get("_courseStyle"));
```

Comment:
Is `_courseStyle` referenced elsewhere? I've been unable to locate it in README, example.json, or properties.schema. However, I found [`_htmlClassName`](https://github.com/adaptlearning/adapt_framework/blob/c442ab5f4e91c899cb1a30812ba631be0ce69d26/src/core/schema/course.model.schema#L51)

Reply:
`_htmlClassName` is related to https://github.com/adaptlearning/adapt_framework/issues/2392

Reply:
Thanks, @moloko . Understanding now that `_htmlClassName` is not related to `_courseStyle`. Aside from the issue with `_courseStyle`, I added some comments to https://github.com/adaptlearning/adapt_framework/issues/2392 and logged a PR (https://github.com/adaptlearning/adapt_framework/pull/2552) in case my comments resound.